src/wallet/init.cpp

```diff
@@ -194,7 +194,11 @@ bool VerifyWallets()
     }
 
     if (gArgs.IsArgSet("-walletdir") && !fs::is_directory(GetWalletDir())) {
-        return InitError(strprintf(_("Error: Specified wallet directory \"%s\" does not exist."), gArgs.GetArg("-walletdir", "").c_str()));
+        if(!fs::exists(GetWalletDir())) {
```

Comment:
Suggestion:
```cpp
if (exists) {
    return "not a directory";
}

return "does not exists";
```

Reply:
nit: (in case you touch the pull for other reasons): Add space after `if`.